chia/protocols/protocol_message_types.py

```diff
@@ -81,3 +81,8 @@ class ProtocolMessageTypes(Enum):
 
     # Simulator protocol
     farm_new_block = 65
+
+    # New harvester protocol
+    new_signage_point_harvester = 66
+    request_plots = 67
+    respond_plots = 68
```

Comment:
I imagine those could also all be allcaps

Reply:
They are lowercase because the network protocol uses these to choose the function which is executed